### DISCUSSION TITLE
Suggestion: Possible to get the citations under "related" to "zotero-better-bibtex"?

### DISCUSSION THREAD

**Discussion by Zzaaii** [retorquere/zotero-better-bibtex#2545] [Ideas]
Hi, 

Zotero-better-bibtext is great adds! I enjoyed a lot working it with Obsidian. 
So gradually, I began to wonder that, would you consider getting the citations under "related" in Zotero into “zotero-better-bibtex”? If it's possible, then I think it would be super useful function! No matter by extracting the citation key of that related record or the default title of the related record. I just don't know if it's technically possible. To make what I said clear. I pasted an example under the related tag here: 

![image](https://github.com/retorquere/zotero-better-bibtex/assets/83008105/eaea992f-7bd0-4169-b94d-aced3e50e133)

**Comment by retorquere** [comment]
It would be partially possible, but in what field would you want to have it added, and how would you use the results? What's the purpose in other words?

**Comment by Zzaaii** [comment]
Thanks for the prompt reply! 

In a short word, it's used for showing all related citations in the third-party software. 

I'm using zotero-better-bibtex to import citations into Obsidian, a notebook software.  so now I have the syn exported information includes title, authors, year, abstracts, notes, see, it's already gotten lots of information! I can export that information of one citation into one note page in Obsidian. Since Obsidian has a powerful function of relating other notes, and Zotero supports relating other citations, if zotero-better-bibtex can incorporate this 'related', I think it'll be a fantastic feature, given that it saves time in relating notes/citations one by one in Obsidian. But I'm not sure, if other non-Obsidian user would also like or have this tendency to use this feature. :P

**Reply by retorquere** [reply]
But in what field would you want it added?

**Reply by Zzaaii** [reply]
I see. Adding in the Note field only for exporting. Or create an extra field? I have no idea which one is technically possible and easier. 

Below is the citation plugin that I used to combine `zotero-better-bibtex` and Obsidian: 
[https://github.com/hans/obsidian-citation-plugin](url)

**Reply by retorquere** [reply]
All of those are possible and the exact same amount of work.

**Reply by Zzaaii** [reply]
Haha. Ok, that's good to know. It takes time. No matter you consider incorporating it or not, I appreciate your above response anyway. :)

**Reply by retorquere** [reply]
I can show you how to do it, but I'd need to know what field you want it in.

**Reply by Zzaaii** [reply]
That's very nice of you! Thanks for your offer. I'll come back for this later tomorrow or in the weekend, currently I'm working for a proposal which has a deadline of tomorrow. 😅

**Reply by Zzaaii** [reply]
Hi, I come back to this topic. Very sorry for the late reply.. I don't understand well the field you mentioned, can you give some examples? As wrote earlier, I thought about creating an extra field, which in this case, what I imaged is: links of citation under 'related' part in Zotero would be exported in an independent field after other texts in the .bib file. Is that so?

**Reply by retorquere** [reply]
I need to know what name you want that independent field to have.

**Comment by Zzaaii** [comment]
I see 😅😂, the name can be "related". This is good enough.
